lib/create-testing-library-rule/index.ts

```diff
@@ -11,24 +14,25 @@ type CreateRuleMeta<TMessageIds extends string> = {
   docs: CreateRuleMetaDocs;
 } & Omit<TSESLint.RuleMetaData<TMessageIds>, 'docs'>;
 
-export function createTestingLibraryRule<
+export const createTestingLibraryRule = <
```

Comment:
Why changing this from regular function to arrow function? Could we please avoid this sort of personal preference? They don't solve anything and add noise to the PR. We could even spend the whole life switching from one definition way to the other every time someone else modifies this file 😓